packages/zoe/src/zoeService/internal-types.js

```diff
@@ -6,7 +6,7 @@
  * @callback CreatePurse
  * @param {Issuer} issuer
  * @param {Brand} brand
- * @returns {void}
+ * @returns {Promise<void>}
```

Comment:
```suggestion
 * @returns {ERef<void>}
```
allows the `return undefined;` in suggested change to `createPurse` above.